muesli/web/templates/lecture/view.pt

```diff
@@ -48,9 +48,12 @@
           <a tal:attributes="href request.route_path('tutorial_view', tutorial_ids=tutorial.id)">Details</a>
         </td>
         <td>
-          <a tal:condition='tutorial.student_count &lt; tutorial.max_students '
+          <a tal:condition='(tutorial.student_count &lt; tutorial.max_students) and tutorial.id != subscribed_tutorial_id'
              tal:attributes="href request.route_path('tutorial_subscribe', tutorial_id=tutorial.id)">
-             ${'Wechseln' if subscribed else 'Beitreten'}
+             ${'Wechseln' if subscribed_to_any_tutorial else 'Beitreten'}
```

Comment:
Can't you use something like `if not subscribed_tutorial_id else`? You would only need one variable. I also prefer renaming this to `subscribed_tutorial` and accessing the id as `subscribed_tutorial.id`, if necessary.

Reply:
Yes, good idea, thank you!